mod/build.gradle.kts

```
@@ -274,7 +274,7 @@ tasks {
         enabled = false
     }
     named<RemapJarTask>("remapJar") {
-        archiveBaseName.set("Skytils")
+        archiveBaseName.set(shadowJar.flatMap { it.archiveBaseName })
```

```suggestion
        archiveBaseName.set(shadowJar.flatMap(ShadowJar::archiveBaseName))
```
I think this works but idk what the type is so hopefully this works